features/features.go

```diff
@@ -40,7 +40,7 @@ var (
 					reportProblemsToJiraComponent("Management Console").
 					contactPerson("jhadvig").
 					productScope(ocpSpecific).
-					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+					enableIn(configv1.Default, configv1.OKD, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Is there ever a reason we would want to promote a feature to Default and not have it also in the OKD featureset? If there isn't, we should either make it so that the OKD featureset inherits from the Default feature set, or, add some sort of test that will flag when something is promoted without OKD being added

Reply:
> How certain are we that every OCP feature will always land in OKD?

100% i would say. Every feature in the default featureset is also in OKD by default.